README.md

```diff
@@ -4,32 +4,30 @@ Aqovia VisualStudioCode ExtensionPack is an extension pack which contains extens
 
 Current extensions list -
 <!-- EXTENSION_PACK_START -->
-[
-  "hnw.vscode-auto-open-markdown-preview",
-  "redhat.vscode-yaml",
-  "Zignd.html-css-class-completion",
-  "streetsidesoftware.code-spell-checker",
-  "ms-vscode.csharp",
-  "donjayamanne.git-extension-pack",
-  "eriklynd.json-tools",
-  "dbaeumer.vscode-eslint",
-  "mrmlnc.vscode-less",
-  "DavidAnson.vscode-markdownlint",
-  "ms-mssql.mssql",
-  "leizongmin.node-module-intellisense",
-  "christian-kohler.npm-intellisense",
-  "polymer.polymer-ide",
-  "ms-vscode.PowerShell",
-  "ms-python.python",
-  "pflannery.vscode-versionlens",
-  "DotJoshJohnson.xml",
-  "bierner.lit-html",
-  "msazurermtools.azurerm-vscode-tools",
-  "ms-azure-devops.azure-pipelines",
-  "mads-hartmann.bash-ide-vscode",
-  "42crunch.vscode-openapi",
-  "mermade.openapi-lint",
-  "adisreyaj.swagger-snippets",
-  "arjun.swagger-viewer"
-]
-<!-- EXTENSION_PACK_END -->
+- hnw.vscode-auto-open-markdown-preview
```

Comment:
Do we need any content between `<!-- EXTENSION_PACK_END -->` and `<!-- EXTENSION_PACK_END -->`?

Reply:
I'm not sure I understand. If we don't have any content the README will not have a list of extensions included in the pack which you requested.